database/index.js

```diff
@@ -28,16 +28,24 @@ const bizSchema = mongoose.Schema({
 const Biz = mongoose.model('Biz', bizSchema);
 
 const photoSchema = mongoose.Schema({
-  pid: Number,
+  pId: Number,
   imgUrl: String,
-  userName: String,
-  userAv: String,
+  uId: Number,
+  bId: Number,
   text: String,
   tag: String,
 });
-const photo = mongoose.model('photo', photoSchema);
+const Photo = mongoose.model('Photo', photoSchema);
+
+const userSchema = mongoose.Schema({
```

Comment:
Same with uID, as the bID. In order to decrease complexity we may want to ultimately keep our self generated id's along with Mongoose's id's. Otherwise our databases would be out of sync. We can discuss in the next meeting, though i suspect generating our own Id's 1-100 will allow for us to spend more time building out Front-
End Features

Reply:
It surely needs the auto increment for generating id! Let's figure it out maybe wail til we have some works done on UI.